airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py

```diff
@@ -138,6 +159,9 @@ class GoogleAnalyticsDataApiBaseStream(GoogleAnalyticsDataApiAbstractStream):
 
     metadata = MetadataDescriptor()
 
+    def get_error_mapping(self) -> Mapping[Union[int, str, Type[Exception]], ErrorResolution]:
+        return get_google_analytics_data_api_base_error_mapping(self.name)
```

Comment:
d.- get_error_mapping is implemented in **GoogleAnalyticsDataApiBaseStream**. 

Previously, check_connection in **SourceGoogleAnalyticsDataApi** would format a config error (WRONG_CUSTOM_REPORT_CONFIG) for HTTP errors checking the connection (read records). Now method _get_google_analytics_data_api_base_error_mapping_ handles this task.